build-support/go-offline.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
this stuff will be wired into circleci like.. https://github.com/openzipkin/zipkin-aws/blob/master/circle.yml